src/actions/pause/tests/pauseOnExceptions.spec.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+import {
+  actions,
+  createStore,
+  getTelemetryEvents
+} from "../../../utils/test-head";
+
+describe("pauseOnExceptions", () => {
+  it("should track telemetry for pauseOnException changes", async () => {
+    const { dispatch } = createStore({ pauseOnExceptions: () => {} });
+    dispatch(actions.pauseOnExceptions(true, false));
```

Comment:
maybe we should also have a test for the redux selectors as well?

Reply:
Could you expound?  Which selectors?

Reply:
```js
expect(shouldPauseOnExceptions(store.getState()).toBe(true);
expect(shouldPauseOnExceptionsOnUncaught(store.getState()).toBe(true);
```

The reason for this is it looks like it is our first POE test.